### ISSUE TITLE
Define a default layout or reorder the layouts sequence

### ISSUE THREAD

**Issue by lyndhurst** [esjeon/krohnkite#114]
Hi, I have been testing the script on neon 5.19, and everything has been working well so far.

The reason I am looking for a script instead of installing a tiling window manager is that my use case makes it really useful only for certain activities like coding or research and writing for example.

For what I was able to observe, the first layout enabled in the configuration is the one used by default, they are then cycled through in that exact order. I would like 'floating' to be selected by default (or tiling disabled), and be able to quickly tile windows (or enable tiling).

Is it possible to define the default tiling layout? Or can I hack into a config file and manually change the order of the enabled layouts sequence?

Thank you for your time, and this great script.

**Comment by lyndhurst** [comment]
I was thinking that maybe as a workaround, there is a command / script I could use to set a particular layout to each of my virtual desktops (they are fixed) as a shortcut, or a cron job on startup ...

**Comment by lyndhurst** [comment]
If I understood correctly this time, the 'Layouts' check boxes in the config menu do not define enabled layouts as I thought, but layouts included in the layout rotation commands.

So it seems I can achieve the desired behavior I described above by checking only the 'floating' layout box in the config (or any layout I would like as default for that matter), and using the layout shortcuts to tile windows when needed.

It seems to work so far; only downside is that I lose the (efficient) kwin ability to remember your preferred window sizes. A tedious workaround I found for now is to set the desired window size in the application special settings as 'apply initially'. The size is being applied when switching back to 'Floating layout'.

I still have a last related question after which this issue can be closed. Is there a way to have window placement rules like 'minimum overlap' applied when reverting back to 'Floating all'?
